Gemfile.lock

```diff
@@ -237,6 +237,7 @@ GEM
       activerecord (>= 5.a)
       database_cleaner-core (~> 2.0.0)
     database_cleaner-core (2.0.1)
+    date (3.3.3)
```

Comment:
I tried rebasing this to resolve the conflict. Rather than manually resolve it, I decided to let Bundler's `install` regenerate the lock file. 

But oddly, it doesn't include this line: `date (3.3.3)`. Even more oddly, if I manually add it back, Bundler's `install` doesn't remove it. It seems to ignore it, so I'm confused about why it's there and if if should be?!

Reply:
@mkllnk or @jibees do you have any idea why?

Reply:
Absolutely not. 
But I'm wondering why this PR has so many updates: `concurrent-ruby`, `bindata` which aren't need by `omniauth_openid_connect` right? Or I missed something.

Reply:
Yes, I also just found this line:
https://github.com/openfoodfoundation/openfoodnetwork/blob/2ae7137dbf7231cd5d86a85992bf59866f2d7374/Gemfile.lock#L420-L421

It should be there. I'll try to regenerate again.